commands/calc.js

```diff
@@ -0,0 +1,45 @@
+var math = require('mathjs');
+const Discord = require("discord.js");
+
+exports.run = (bot, message, args, command) => {
```

Comment:
Please follow the variable labelling of the project, we do not use `bot`, we use `client`